packages/react/src/components/TextField/TextField.types.ts

```diff
@@ -278,6 +278,14 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
    */
   canRevealPassword?: boolean;
 
+  /**
+   * Whether to use the default password reveal button. This only applies to Edge and IE and using the default
+   * reveal icon will show both IE/Edge's button and Fluent UI's button.
+   *
+   * @defaultvalue false
+   */
+  enableBuiltInRevealButton?: boolean;
```

Comment:
I'd lean towards renaming this to be clearer -- "built-in" could refer to either a button built into Fluent, or built into the browser (I actually initially thought it meant the opposite of what it does 😅). Maybe something like "enableBrowserPasswordReveal"?